sdk/netapp/Microsoft.Azure.Management.NetApp/src/Microsoft.Azure.Management.NetApp.csproj

```diff
@@ -13,8 +13,8 @@
     <PackageTags>MicrosoftAzure Management;NetApp</PackageTags>
     <PackageReleaseNotes>
       <![CDATA[
-        Version 1.5.0 relates to NetApp Files (ANF) RP R8 GA.
-          - added operation to revert volume to snapshot 
+        Version 1.5.1 relates to NetApp Files (ANF) 2019-11-01.
```

Comment:
Hi, I think you need to bump up to 2.0.0 for the breaking change, unless of course the previous version wouldn't work at all.
Besides, please also update the <Version>1.5.0</Version> in this file

Reply:
I was not sure about that since this is a bugfix for a broken functionality. I am happy to bump to 2.0. if you prefer.

Reply:
@isra-fel  updated